include/dxc/Support/HLSLOptions.td

```diff
@@ -405,6 +405,8 @@ def P : Separate<["-", "/"], "P">, Flags<[CoreOption, DriverOption]>, Group<hlsl
 
 def dumpbin : Flag<["-", "/"], "dumpbin">, Flags<[DriverOption]>, Group<hlslutil_Group>,
   HelpText<"Load a binary file rather than compiling">;
+def link : Flag<["-", "/"], "link">, Flags<[DriverOption]>, Group<hlslutil_Group>,
+  HelpText<"Linking rather than compiling">;
```

Comment:
Help text could be better, it doesn't say how to use the -link option and provide multiple file input.
Perhaps:
`Link list of libraries provided in <inputs> argument separated by ';'`